### PR TITLE
Use lined paper vector and remove unnecessary nav

### PR DESCRIPTION
**Pull request by waalbert**
The lined paper background of the navigation bar has been switched to use the vector version. The unnecessary nested `nav` element has been changed to a `div`

### CODE REVIEW

File: apps/site/src/components/NavBar/NavBar.tsx [hunk 1]
```diff
@@ -16,7 +16,7 @@ export default function NavBar() {
 	const activeRoute = usePathname();
 
 	return (
-		<nav className={styles.nav}>
+		<div className={styles.nav}>
```
Comment (taesungh) [inline apps/site/src/components/NavBar/NavBar.tsx:19]: Thought: do we even need this parent element?

Comment (waalbert) [reply apps/site/src/components/NavBar/NavBar.tsx:19]: I originally added the parent element to implement the functionality of the ZotHacks logo moving down with the nav bar drop-down. I put the ZotHacks logo as a sibling of the rest of the nav bar to allow for this

Comment (taesungh) [reply apps/site/src/components/NavBar/NavBar.tsx:19]: I think the sibling behavior is possible with just a fragment, but I suppose the parent styles are necessary if we want sticky behavior, but that doesn't seem to be working right now. Could we reconfirm with graphics what the desired behavior is?

**Review by taesungh — APPROVED** [review]
Works out nicely.

**Review by samderanova — APPROVED** [review]
Looks good, thanks for working on this!

**Review by taesungh — APPROVED** [review]

**Pull request merged**